### ISSUE TITLE
Can't inject minified jquery

### ISSUE THREAD

**Issue by chentsulin** [segmentio/nightmare#323]
This works great:

```
nightmare.inject('js', `node_modules/jquery/dist/jquery.js`);
```

but minified file not:

```
nightmare.inject('js', `node_modules/jquery/dist/jquery.min.js`);
```

Any ideas?

**Comment by TonyWang031** [comment]
Got the same issue too.

**Comment by matthewmueller** [comment]
can you narrow down where the error might be occuring?

**Comment by rosshinkley** [comment]
@chentsulin @TonyWang031 Depending on the version of jQuery being injected, they could end with sourcemap comments, which would cause the end of the calling function that encapsulates the injected script to be commented out, causing a runtime error.  I would be willing to bet this is what you were hitting, given the unminified version worked but the minified version wouldn't.

The template was fixed in #524, so I am going to go ahead and close this issue.  If you still have problems, feel free to reopen/submit a new one.
